tests/test_client_response.py

```diff
@@ -453,35 +454,7 @@ def side_effect(*args, **kwargs):
     res = await response.text()
     assert res == '{"тест": "пройден"}'
     assert response._connection is None
-
-
-async def test_text_detect_encoding_if_invalid_charset(loop: Any, session: Any) -> None:
-    response = ClientResponse(
-        "get",
-        URL("http://def-cl-resp.org"),
-        request_info=mock.Mock(),
-        writer=mock.Mock(),
-        continue100=None,
-        timer=TimerNoop(),
-        traces=[],
-        loop=loop,
-        session=session,
-    )
-
-    def side_effect(*args, **kwargs):
-        fut = loop.create_future()
-        fut.set_result('{"тест": "пройден"}'.encode("cp1251"))
-        return fut
-
-    response._headers = {"Content-Type": "text/plain;charset=invalid"}
```

Comment:
This one tests when charset=invalid, which we probably still want to test. Though you could probably just parametrize the above test, so it tests with no charset and charset=invalid.

Reply:
Yeah, I see that. In the docs I'm updating I keep having to say "no charset, or a charset that python doesn't understand."

Reply:
@Dreamsorcerer I parameterized the test. I haven't done that before, so do make sure I did it correctly.